orchestra/models/core/mixins.py

```diff
@@ -174,6 +174,23 @@ def is_entry_level(self):
     def is_reviewer(self):
         return self.assignment_counter > 0
 
+    def _can_view_tag(self, tag, certifications):
+        for tag_certification in tag.get('certificates', []):
```

Comment:
verifying tags has no connection tot he assignment, so this is more like get_certification_tags.

i think we should call the function `get_task_tags` and verify against the step slug.